salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -1,7 +1,12 @@
 {%- set node_name = pillar.orchestrate.node_name %}
 {%- set version = pillar.metalk8s.nodes[node_name].version %}
 
-{%- set skip_roles = pillar.metalk8s.nodes[node_name].get('skip_roles', []) %}
+{%- set roles_to_skip = pillar.metalk8s.nodes[node_name].get('skip_roles', []) %}
+
+{#- Add etcd to skip_roles in order not to apply all the highstate #}
+{%- set skip_roles = roles_to_skip.append('etcd') %}
```

Comment:
`append` do not return the list it's just alter the list so you can't do this as `skip_roles` will always be empty